webpack.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
         include : SRC_DIR,
         loader : 'babel-loader',      
         query: {
-          presets: ['react', 'es2015']
+          presets: ['react', 'es2015','stage-0']
```

Comment:
Just curious, what does stage-0 do?

Reply:
reads the spread operator i'm using when changing specific properties of the newVolunteer state

Reply:
yes, they are supposed to be reusable. ill create a folder for elements.